apps/codeforafrica/src/assets/twitterMobile.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="27" viewBox="0 0 32 27" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We don't need this file. We already have this [icon](https://github.com/CodeForAfrica/ui/blob/main/apps/codeforafrica/src/assets/icons/Type%3Dtwitter%2C%20Size%3D24%2C%20Color%3DCurrentColor.svg). You only need to set `color` to white/text secondary on `Link` component surrounding the `SvgIcon` for the icon to appear _white_. https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/color